1-js/02-first-steps/09-comparison/article.md

```diff
@@ -112,7 +112,7 @@ From JavaScript's standpoint, this result is quite normal. An equality check con
 
 ## Strict equality
 
-A regular equality check `==` has a problem. It cannot differentiate `0` from `false`:
+A regular equality check `==` has a problem. For example, it cannot differentiate `0` from `false`:
```

Comment:
Please remove "for example", it doesn't fit the phrase.

Reply:
I will soon. But from the current wording, would not a reader think the 0\false issue is the only issue with ==?

Reply:
We're talking about this particular problem here. 

Maybe you'd want to further modify the statement. Right now, "For example" just doesn't fit.

Reply:
Ok.

Reply:
Ok.

Reply:
Ok.

Reply:
https://www.dictionary.com/browse/for-example :
> This program has problems—for instance, it's hard to retrieve lost data.

"For instance" and "for example" can be used interchangeably. But "for instance" may sound better in this case.

Reply:
Change excluded,